recipe/build.sh

```diff
@@ -109,7 +75,9 @@ done
 sed -i.bak "s@./build-normaliz.sh@echo@g" ../bin/BuildPackages.sh
 bash ../bin/BuildPackages.sh \
    --add-package-config-Semigroups "--with-external-libsemigroups --without-march-native" \
-   --add-package-config-Digraphs "--with-external-bliss --with-external-planarity --without-intrinsics"
+   --add-package-config-semigroups "--with-external-libsemigroups --without-march-native" \
+   --add-package-config-Digraphs "--with-external-bliss --with-external-planarity --without-intrinsics" \
+   --add-package-config-digraphs "--with-external-bliss --with-external-planarity --without-intrinsics"
```

Comment:
So, instead of using `BuildPackages.sh` you could also execute something like this in a loop over all packagesubdirs (you probably need to adjust / replace `$GAPROOT`):
```
for pkdir in SOMETHING_I_AM_TOO_LAZY_TO_FIGURE_OUT ; do
  pushd $pkgdir
  if [ "x$pkgdir" = xdigraphs ] ; then
    ./configure --with-gaproot="$GAPROOT" --with-external-bliss --with-external-planarity --without-intrinsics
  elif [ "x$pkgdir" = xsemigroups ] ; then
    ./configure --with-gaproot="$GAPROOT" --with-with-external-libsemigroups
  elif [ -x configure ] ; then
    # TODO: with GAP 4.13, it will be possible to just use the --with-gaproot form
    # all packages with a `configure` script will then support it
    if grep Autoconf ./configure > /dev/null ; then
      ./configure --with-gaproot="$GAPROOT"
    else
      ./configure "$GAPROOT"
    fi
    make
  fi
done
```

BTW the --without-march-native should be default these days for semigroups.

If you go this way, then you don't need to patch `BuildPackages.sh` either.